### ISSUE TITLE
Below the fold images total size

### ISSUE THREAD

**Issue by ousamabenyounes** [YellowLabTools/YellowLabTools#336]
I am happy when I see that a website's partner is not using the lazy loading feature.
I think it should be interresting to tell how many size the website will be earning using it 

my website is 3,5 mo without lazy loading images. It will be 2,5 mo if i enable this really important feature.

Also in the images listing, we should be able to see one by one the image's size?

Regards,
Ousama.

**Comment by gmetais** [comment]
Nice idea! Would it be on the audit details page or directly in the main results dashboard?

Shouldn't we do the same for "Hidden images"?

**Comment by ousamabenyounes** [comment]
I agree, I would love to have on the top the website size before easy optmization
and after ....(lazy loading, image optimization, gzip...)
That would be really interresting to see it on the beggining, on the top

A really to fix area:
You site is 4mo today and you have X HttpRequest, 
just enabling lazyloading and image optimization your site will be 2mo with X HttpRequest 
So you can easily earn Xmo and X unusefull requests...


Regards

**Comment by ozcoder** [comment]
There are a few way to get those images. Example:
```pictures = document.getElementsByTagName('img');
